apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -73,7 +71,7 @@ const ContributionsTable = ({
   nextPage,
 }: {
   contributionsData: UIContribution[][];
-  setSelectedContributions: (rows: Row<ContributionTableType>[]) => void;
+  setSelectedContributions: (rows: UIContribution[]) => void;
```

Comment:
So nice! Having the separate types for this led to a lot of additional complexity